### PR TITLE
Implement correct transition behavior on homepage carousel slider

### PR DESCRIPTION
**Pull request by kadamwhite**
This PR implements the specified transition behavior as closely as possible given the time available. 
Bootstrap's carousel was insufficient for this kind of complex transition, so this reimagines `split-carousel.js` as `header-carousel.js` and implements a custom carousel module.

Small:

small

Medium & up:

medium

Note that the quick links for slides, unless you're proceeding to the next slide in the set, cuts directly to that target slide rather than transitioning (as the correct preview image will not be present, so the full transition is impossible). Auto-advance is also not currently implemented, due to time constraints.

Note that slide alignment should be somewhat controllable as established in #113, but this has not been thoroughly tested as the sample images are compositionally similar.

### CODE REVIEW

File: home.html [hunk 1]
```diff
@@ -5,9 +5,15 @@
   <title>Home :: Greenpeace 2017</title>
   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
   <link rel="shortcut icon" type="image/ico" href="favicon.ico" />
+  <!--
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
+  -->
```
Comment (comzeradd) [inline home.html]: We could completely remove these line, since we use local files instead.

File: home.html [hunk 2]
```diff
@@ -5,9 +5,15 @@
...
+  <link rel="stylesheet" href="vendor/bootstrap.min.css">
   <link rel="stylesheet" type="text/css" href="style.css">
+  <!--
   <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.min.css">
+  -->
```
Comment (comzeradd) [inline home.html]: Same here.

File: home.html (hunk 1 above)
Comment (koyan) [reply home.html]: Currently we are using the following (on the site): https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/js/bootstrap.min.js
( https://github.com/greenpeace/planet4-master-theme/blob/develop/functions.php#L278-L286 )
Do we need to switch to the -beta on the master theme?

File: home.html (hunk 2 above)
Comment (koyan) [reply home.html]: Currently using this on the master theme: 
https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.min.js
So, no need to do any changes on master theme.

File: home.html [hunk 3]
```diff
@@ -228,72 +234,68 @@ <h3 class="country-group-letter">U</h3>
...
+                <p>Marshmallow tootsie roll sugar plum powder oatcake ginger bread. Tart liquorice sweet roll pastry croissant toffee mufﬁn tiramisu. Cupcake cookie cotton candy chocolate cake oat ice sweet roll. </p>
                 <div class="col-sm-12 col-md-4 col-lg-4 action-button">
                   <a href="#" class="btn btn-small btn-medium btn-primary btn-block">Take action</a>
                 </div>
               </div>
             </div>
           </div>
         </div>
-        <a class="carousel-control-prev hidden-sm-down" href="#carousel-wrapper-header" role="button" data-slide="prev">
+        <!-- .carousel-control-prev is display: none, so may be omitted from HTML if desired -->
```
Comment (comzeradd) [inline home.html]: I'd remove this block completely, since we don't show the "previous" control.

File: home.html [hunk 4]
```diff
@@ -606,10 +608,16 @@ <h2>Get action alerts in your inbox</h2>
   <!-- Cookie Block End -->
 
 <!-- JS Libs that we include -->
+<!--
 <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.js"></script>
 <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.11.0/umd/popper.min.js" integrity="sha384-b/U6ypiBEHpOf/4+1nzFpr53nxSS+GLCkfwBdFNTxtclqqenISfwAzpKaMNFNmj4" crossorigin="anonymous"></script>
 <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/js/bootstrap.min.js" integrity="sha384-h0AbiXch4ZDo7tp9hKZ4TsHbi047NrKGLO3SEJAg45jXxnGIfYzk4Si90RDIqNm1" crossorigin="anonymous"></script>
 <script src="https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.min.js"></script>
+-->
```
Comment (comzeradd) [inline home.html]: Same as css imports, we can remove these lines.

File: src/scss/common/_carousel-header.scss [hunk 5]
```diff
@@ -123,18 +314,16 @@
...
 
-@include small-and-up {
-  .split-carousel-wrap.carousel-header {
+  @include small-and-up {
     .carousel-item {
       .carousel-caption {
-        height: 100%;
-        padding: 0;
+        // height: 100%;
+        // padding: 0;
```
Comment (comzeradd) [inline src/scss/common/_carousel-header.scss]: Let's remove these two lines if we don't need them anymore.

File: home.html (hunk 1 above)
Comment (kadamwhite) [reply home.html]: Whoops, this was debugging code, I added the local files just to make it work while I was on a spotty connection. This entire part of the PR can be ignored, it has no bearing on the carousel code.

**Comment by kadamwhite** [comment]
Thanks for the review, y'all, sorry to have left things a bit untidy in here! :raised_hands:

**Pull request merged**